scipy/stats/tests/test_multivariate.py

```diff
@@ -2003,11 +2005,24 @@ def test_frozen_distribution(self):
 
         rvs1 = frozen.rvs(random_state=514)
         rvs2 = random_direction.rvs(dim, random_state=514)
-        rvs3 = frozen_seed.rvs(size=1)
+        rvs3 = frozen_seed.rvs()
```

Comment:
The difference between `size=1` and `size=None` is that `size=None` returns a 1-D array (`.shape == (dim, )`) whereas `size=1` returns a 2D array with a singleton dimension (`.shape == (1, dim)`). 

```python3
rng.multivariate_normal(mean=np.zeros(3), cov=np.eye(3))  # array([ 1.31948167, -0.31018314, -1.8922078 ])
rng.multivariate_normal(mean=np.zeros(3), cov=np.eye(3), size=1)  # array([[-0.3628523 , -0.63526532,  0.43181166]])
```

But right now:
```python3
random_direction(3).rvs().shape  # (1, 3)
```

Reply:
Done acc. to your review.